packages/expo-image-picker/ios/MediaHandler.swift

```diff
@@ -130,6 +135,80 @@ internal struct MediaHandler {
     )
   }
 
+  /**
+   * Unlike the case of regular images, we have to operate on original data of the image in order to preserve the exif data,
+   * otherwise it won't be possible to connect the image and video into a `PHLivePhoto` after reading it from the cache directory later.
+   * As a result a live photo photo cannot be compressed or edited.
+   */
```

Comment:
This comment sounds more like something that should be inline comment. Use `/** */` comments only for documenting what this specific function does without going into implementation details. Inline comments are better to describe implementation details and decisions we've made.
Also, the doc comment should always be in the third person form.